### ISSUE TITLE
New OSTs cannot be downloaded

### ISSUE THREAD

**Issue by VulpesJack** [obskyr/khinsider#15]
KHInsider has changed the way they parse URLs for new OSTs. Instead of the old format (e.g. 1-track1.mp3), it now uses percent-encoding (e.g. 101%2520-%2520Wake%2520Up%252C%2520Get%2520Up%252C%2520Get%2520Out%2520There.mp3).

Trying to download an OST that uses this format throws up an error which I've attached as a screenshot.

![image](https://user-images.githubusercontent.com/24865767/31067880-2dbf702e-a74d-11e7-9ab3-4c44166bde27.png)

**Comment by VulpesJack** [comment]
Okay, so I seemed to have fixed it. I commented out line 131, and it seems to be downloading the files just fine and they play.

![image](https://user-images.githubusercontent.com/24865767/31068028-e556924e-a74d-11e7-9e81-f5ba67e1f202.png)

**Comment by Tantusar** [comment]
Sorry, which line 131 is that? The one at my end reads:
```python
        name = name.split('-', 1)[1]
```

**Comment by obskyr** [comment]
Thanks for the heads-up! I'll get on this shortly. Might take a *bit* more to make it work with the search feature and all that.

**Comment by Tantusar** [comment]
I'm actually getting an entirely different NoneIndex error.

**Comment by Tantusar** [comment]
![image](https://user-images.githubusercontent.com/5593775/31110911-fb68dbec-a855-11e7-90c1-592e112e6604.png)

**Comment by obskyr** [comment]
@Tantusar Looks like they may have changed the HTML of the search page in addition to the URL structure. You might be able to grab the soundtrack ID directly from the KHInsider page.

I should probably add more user-friendly error messages for that kind of thing... 😅

**Comment by VulpesJack** [comment]
@Tantusar @obskyr yeah line 131 is the `name = name.split` line. I just simply added a `#` before it and the program seems to function fully, including the search features.

![image](https://user-images.githubusercontent.com/24865767/31112630-b44c53e0-a80c-11e7-9d25-ef9d175ed2c6.png)

The files that I've downloaded using are still named properly, and play just fine.

![image](https://user-images.githubusercontent.com/24865767/31112660-e6cda576-a80c-11e7-8711-4e0d233ea623.png)
